### PR TITLE
Test improvements

### PR DESCRIPTION
**Pull request by mboersma**
Fixes keys tests and remaining auth tests. Also shortens the essential env var to `DEIS_HOST` and hides example app directories under _tests from version control.

### CODE REVIEW

File: _tests/tests_suite_test.go [hunk 1]
```diff
@@ -118,24 +121,23 @@ func addKey(name string) {
...
 	_, err := execute("eval $(ssh-agent) && ssh-add %s", path)
 	Expect(err).NotTo(HaveOccurred())
-	// add the public key to deis (assumes the user is logged in)
-	_, err = execute("deis keys:add %s.pub", path)
-	Expect(err).NotTo(HaveOccurred())
 }
 
 func getController() string {
-	host := os.Getenv("DEIS_WORKFLOW_SERVICE_HOST")
+	host := os.Getenv("DEIS_HOST")
```
Comment (arschles) [inline _tests/tests_suite_test.go]: why the change here and below?

Comment (sgoings) [inline _tests/tests_suite_test.go]: Less typing? :smile:

Comment (arschles) [inline _tests/tests_suite_test.go]: it would be nice to keep this the same as the service env var so that this could easily be run in a pod...

**Comment by arschles** [comment]
closed in favor of https://github.com/deis/workflow/pull/184
